flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/DispatcherTest.java

```diff
@@ -1158,6 +1162,102 @@ public void testRequestMultipleJobDetails_returnsFinishedOverSuspendedJob() thro
                 JobStatus.FINISHED, dispatcherGateway.requestMultipleJobDetails(TIMEOUT).get());
     }
 
+    @Test
+    public void testRequestMultipleJobDetails_returnsJobsOfSameStateOrderedByStartTimeInDecOrder()
```

Comment:
nit: The newly added tests have the almost the same code. We could move the generic code out into a separate method and just define parameters for input (the JobManagerRunners and expected JobID order). WDYT?

Reply:
yes, that would ease reading through them as well.